### PR TITLE
[rum] Update Expo View Tracking docs

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do? What is the motivation?

As per [DOCS-8088](https://datadoghq.atlassian.net/browse/DOCS-8088), this update adds instructions on how to add View Tracking for Expo. 

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

[DOCS-8088]: https://datadoghq.atlassian.net/browse/DOCS-8088?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by hestonhoffman — APPROVED** [review]
small edit

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/expo.md [hunk 1]
```diff
@@ -46,6 +46,49 @@ To install with Yarn, run:
 yarn add expo-datadog @datadog/mobile-react-native
 ```
 
+### Track view navigation
+
+To see RUM sessions populate in Datadog, you need to implement view tracking, which can be initialized manually or automatically.
+
+#### Manual initialization
```
Comment (marco-saia-datadog) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/expo.md:53]:
```suggestion
#### Manual tracking
```

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/expo.md [hunk 2]
```diff
@@ -46,6 +46,49 @@ To install with Yarn, run:
...
+    '<view-key>', // <view-key> has to be unique, for example it can be ViewName-unique-id
+    'View Name',
+    { 'custom.foo': 'something' },
+    Date.now()
+);
+// Stops a previously started view with the same unique view identifier, and an object to attach additional attributes to the view
+DdRum.stopView('<view-key>', { 'custom.bar': 42 }, Date.now());
+```
+
+#### Automatic initialization
```
Comment (marco-saia-datadog) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/expo.md:73]:
```suggestion
#### Automatic tracking
```

**Review by marco-saia-datadog — CHANGES_REQUESTED** [review]

**Review by MaelNamNam — APPROVED** [review]
LGTM! Thanks chief!

**Review by marco-saia-datadog — APPROVED** [review]
Looks good to me 👍

**Pull request merged**